x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/discover/discover_state.cy.ts

```diff
@@ -33,19 +29,17 @@ const INITIAL_END_DATE = 'Jan 19, 2024 @ 20:33:29.186';
 const DEFAULT_ESQL_QUERY =
   'from .alerts-security.alerts-default,apm-*-transaction*,auditbeat-*,endgame-*,filebeat-*,logs-*,packetbeat-*,traces-apm*,winlogbeat-*,-*elastic-cloud-logs-* | limit 10';
 
-// TODO: reuse or remove this tests when ESQL tab will be added
-describe.skip(
```

Comment:
We should rename this test as well as the file and some locators to be aligned with the new reality.

Reply:
Done